src/pages/components/utilities.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Utilities
+description: >
+  Utility functions enable advanced theme usage. They let you customize
+  components in a granular way.
+package: '@zendeskgarden/react-theming'
+components:
+  - theming/src/utils/arrowStyles.ts
+  - theming/src/utils/getColor.ts
+  - theming/src/utils/getLineHeight.ts
+  - theming/src/utils/mediaQuery.ts
+  - theming/src/utils/menuStyles.ts
+  - theming/src/utils/retrieveComponentStyles.ts
+  - theming/src/utils/withTheme.ts
+---
+
+## How to use it
```

Comment:
Should we lose this heading since there is nothing underneath it?

Reply:
No. This is standard throughout the component pages and we must retain a valid H1 -> H2 -> H3 for accessibility.

Reply:
Although, should this be "How to use them" as we are referring to Utilities rather than a single Component?